azext_iot/digitaltwins/params.py

```diff
@@ -164,6 +179,19 @@ def load_digitaltwins_arguments(self, _):
             options_list=["--auth-type"],
             help="Endpoint authentication type.",
             arg_type=get_enum_type(ADTEndpointAuthType),
+            deprecate_info=context.deprecate(redirect="identity")
+        )
+        context.argument(
+            'system_identity',
+            options_list=['--mi-system-assigned', '--system'],
+            arg_type=get_three_state_flag(),
+            help="Use a system-assigned managed identity for endpoint authentication."
+        )
+        context.argument(
+            'user_identity',
+            options_list=['--mi-user-assigned', '--user'],
+            help="Use an user-assigned managed identities for endpoint authentication. "
+            "Accepts the identity resource ID."
```

Comment:
ID in resource ID is not always capitalized, lets use consistent casing

Reply:
![image](https://user-images.githubusercontent.com/28658112/199842438-64d52ff3-1b19-444b-a9e6-61c23c387d33.png)

Reply:
Going to lowercase all IDs in the param file.